README.md

```diff
@@ -4,12 +4,67 @@ Substrate types used in polymesh chain.
 
 ## Usage
 
-Import the types in your JS or TS app.
+### Basic Usage (Type Bundles)
+
+To use the Polymesh type bundles with the Polkadot.js API:
+
+```ts
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import { typesBundle } from '@polymeshassociation/polymesh-types';
+
+async function main() {
+  const provider = new WsProvider('wss://testnet-rpc.polymesh.live');
+  const api = await ApiPromise.create({
+    provider,
+    typesBundle,
+  });
+  // ... your code here
+}
+```
+
+### Using Type Augmentations
+
+To get full TypeScript support for Polymesh-specific types, import the type augmentations **before** using the API types or instantiating the Polkadot API. The best practice is to place these imports at the very top of your project's entry point (e.g., `src/index.ts` or `src/main.ts`).
+
+If your `tsconfig.json` includes the `src` folder (e.g., `"include": ["src"]`), the type augmentations will be available globally to all files in `src`.
 
 ```ts
+// At the top of your entry point file
+import '@polymeshassociation/polymesh-types/generated-types/augment-types';
+import '@polymeshassociation/polymesh-types/generated-types/augment-api';
```

Comment:
I prefer this path but it doesn't match what you updated the scripts to use so one of them needs to be corrected